viz.yaml

```diff
@@ -215,7 +215,7 @@ fetch:
     id: nws-conversion
     location: cache/nws-conversion.rds
     reader: rds
-    states: ["FL"] # not currently coded to support more...
+    states: ["FL", "GA"] 
```

Comment:
We should figure out how to have one list of states for the whole viz. We could derive this list from the polygon that's passed in for the storm area intersected with a list of states?

Reply:
Yes, that would be good.